.buildkite/pipeline.yml

```
@@ -312,9 +312,9 @@ steps:
   - label: ":computer: Rising Bubble 2D hybrid"
     key: "cpu_rising_bubble_2d_hybrid"
     command:
-      - "julia --color=yes --project=examples examples/hybrid/bubble_2d.jl"
+      - "julia --color=yes --project=examples examples/hybrid/box/bubble_2d.jl"
```

Please consider moving the 2d examples in a separate `examples/hybrid/plane/` directory for consistency with the non-hybrid case.
```suggestion
      - "julia --color=yes --project=examples examples/hybrid/plane/bubble_2d.jl"
```

---

I thought 2d plane is usually a 2d horizontal cg plane, which is different from here

---

It's just a category based on geometry/domain type.  There was a discussion on [Slack](https://climate-machine.slack.com/archives/C019DHELDA9/p1643931125682369) where we talked about characterizing them by physical behavior or by domain type. Since this PR makes a move in the latter direction, I think at least it would be good to follow a consistent pattern

---

So we would have the non-hybrid (CG only examples) in `examples/plane`, and the corresponding hybrid ones in `examples/hybrid/plane` and `examples/hybrid/box`, respectively (for 2D and 3D). It's just clearer and more consistent IMO.

---

Ok, so if we're categorizing things based on domains, perhaps we could have hybrid/line (or hybrid/interval), hybrid/plane (or hybrid/rectangle), and hybrid/sphere (or hybrid/cubed_sphere).

---

In the absence of a broader discussion with the others, I personally think it's ok to leave the `hybrid/box` for the hybrid 3d ones as you did. And just change the `hybrid/plane` ones as I was asking, to make them at least consistent with the planar non-hybrid case. Some of these examples will be moved anyways, so not worth doing a major reorganization of the examples folder right now.

---

Alright, will do

---

Thank you